### PR TITLE
Create Giana's first Poem

### PR DESCRIPTION
**Pull request by GianaG8**

### CODE REVIEW

File: Giana's first Poem [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Roses are red
+Violets are blue
+I love pizza
+But Pineapples are ew
```
Comment (jkschulz) [inline Giana's first Poem]: lowercase `pineapples` please!

**Review by jkschulz — CHANGES_REQUESTED** [review]
Great! One suggestion

**Comment by GianaG8** [comment]
> Great! One suggestion

I'm very confused

**Review by GianaG8 — COMMENTED** [review]
Is this how I do it?

**Review by jkschulz — APPROVED** [review]

**Pull request merged**